lib/serve/rest-api/src/entrypoint.sh

```diff
@@ -5,6 +5,14 @@ set -e
 HOST="0.0.0.0"
 PORT="8080"
 
+# Update LiteLLM config that was already copied from config.yaml with runtime-deployed models.
+# Depends on SSM Parameter for registered models.
+echo "Configuring and starting LiteLLM"
+python ./src/utils/generate_litellm_config.py -f litellm_config.yaml
```

Comment:
Similar to my other comment we should put a comment here about how this file is generated from the `Dockerfile` and the name shouldn't change here unless it changes there.

Reply:
Will do, thanks!